### PR TITLE
Add explanation for the loss function

### PR DESCRIPTION
**Pull request by vwxyzjn**
One of the most confusing things about implementing the A3C paper is that the entropy is not mentioned in the pseudocode at https://arxiv.org/pdf/1602.01783.pdf#page=14. I was really confused about how the loss was defined. Hope this comment will clear things up.

### CODE REVIEW

File: stable_baselines/a2c/a2c.py [hunk 1]
```diff
@@ -127,6 +127,8 @@ def setup_model(self):
                     self.entropy = tf.reduce_mean(train_model.proba_distribution.entropy())
                     self.pg_loss = tf.reduce_mean(self.advs_ph * neglogpac)
                     self.vf_loss = mse(tf.squeeze(train_model._value), self.rewards_ph)
+                    # https://arxiv.org/pdf/1708.04782.pdf#page=9 and https://arxiv.org/pdf/1602.01783.pdf#page=4
+                    # suggests that entropy is used for improving exploration.
```
Comment (araffin) [inline stable_baselines/a2c/a2c.py]:
```suggestion
                    # suggest to add an entropy bonus in order to improve exploration.
```

**Comment by araffin** [comment]
Hello,
thank you for your PR. Small reformulation and we can merge it ;)

**Comment by vwxyzjn** [comment]
@araffin I suggest using the phrasing "suggest to add an entropy component in order to improve exploration". The "entropy bonus" seems a bit odd to me under this context.
